src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const table = document.body.firstElementChild;
+
+for (let i = 0; i < table.rows.length; i++) {
+  const columnToClone = table.rows[i].cells[1];
```

Comment:
You can use `document.querySelectorAll('tr')` to get an array of rows and work with them. This approach will be more efficient in my opinion.